include/cuspv/cuda_runtime.h

```diff
@@ -341,7 +366,6 @@ using cudaDeviceAttribute_t = hipDeviceAttribute_t;
 using cudaDevice_t = hipDevice_t;
 using cudaError_t = hipError_t;
 using cudaEvent_t = hipEvent_t;
-using cudaExtent = hipExtent;
 using cudaFuncAttributes = hipFuncAttributes;
 using cudaFuncCache = hipFuncCache_t;
 using cudaLimit = hipLimit_t;
```

Comment:
What's the reason for mapping here via "using" instead of type redefinition?

Reply:
The `using` is modern version of `typedef`, and more readable in general.

Reply:
Yes so my question was why would we remove `using` in favor of `define`

Reply:
Ok, I misread the question. I don't have recollection for this change.